### PR TITLE
Tabularize the oncall page

### PR DESCRIPTION
**Pull request by amwat**
Also add a postsubmit to upload the changes whenever it changes.

/cc @BenTheElder @spiffxp

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 1]
```diff
@@ -575,6 +575,32 @@ postsubmits:
...
         - --build-dir=.
         - gencred/
+  - name: post-test-infra-upload-oncall
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^maintenance/oncall.html$'
+    decorate: true
+    spec:
+      serviceAccountName: pusher
```
Comment (amwat) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:585]: not sure if this is the right one

Comment (BenTheElder) [reply config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:585]: this is also not going to upload it I think because it's in the same PR

Comment (BenTheElder) [reply config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:585]: we'll need to do a follow-up change to the file. I plan to at some point anyhow

**Pull request merged**